packages/protocol/engine/sybilprotection/dpos/sybilprotection.go

```diff
@@ -102,6 +102,8 @@ func (s *SybilProtection) initializeActiveValidators() {
 	}); err != nil {
 		panic(err)
 	}
+
+	s.TriggerInitialized()
```

Comment:
if this is in here, maybe rename the function to `initialize` ?

Reply:
That file has 3 initialize* doing different setup steps, so not sure it will make it clearer to rename it